syft/workers/base.py

```diff
@@ -186,9 +186,6 @@ def get_obj(self, obj_id: Union[str, int]) -> object:
         """
         return self.object_store.get_obj(obj_id)
 
-    def register_obj(self, obj):
```

Comment:
Q: I think this needs to be put back (I think there was another PR merged with this change)

Reply:
If I've checked properly, the duplicate method still exists.